### DISCUSSION TITLE
question about pgm

### DISCUSSION THREAD

**Discussion by Jocelyn62215** [ParadoxAlarmInterface/pai#449] [General]
Good morning,
I have a question regarding PGMs.
I use PAI to connect home assistant to a paradox evo192 alarm in version 7.70.001. everything works perfectly.... except :)
I can use the pgms from the control panel (it works well) but I cannot use the pgms from the additional modules (keyboards, zx8, pgm4)
Is there a special configuration to be able to manually activate the pgm4 outputs from home assistant for example?
I couldn't find any notice on this subject

Thanks for your help

**Comment by yozik04** [comment]
Hey. Unfortunately PGMs from additional modules are not supported. I do not personally have these, so I was unable to implement.

**Comment by Jocelyn62215** [comment]
thank you for the answer, it's a shame, we already have modules for the alarm throughout the house, it would have been nice to be able to use the outputs, just to avoid radio modules. I already had lots of ideas in mind :)
